charts/neon-proxy/README.md

```diff
@@ -60,7 +60,7 @@ Kubernetes: `^1.18.x-x`
 | settings.authBackend | string | `"link"` | auth method used (console|link|postgres) |
 | settings.authEndpoint | string | `""` | auth endpoint, e.g. "http://console.neon/authenticate_proxy_request/" |
 | settings.domain | string | `""` | domain used in TLS cert for client postgres connections |
-| settings.sentryURL | string | `""` | url used by sentry to collect error/panic events in neon-proxy |
+| settings.sentryUrl | string | `""` | url used by sentry to collect error/panic events in neon-proxy |
```

Comment:
We now have a bit different conventions, as `neon-proxy` uses `sentryURL`, although probably not worth changing. And if changing, maybe using DSN would be the choice to consider to be consisten with the ENV variable and Sentry terminology. So just leaving the comment as a trail.

Reply:
I've made it similar: both containers accept `settings.sentryUrl` that will be transformed into `SENTRY_DSN` env variable. what do you propose to change in order to keep it clean?

Reply:
Ah, so I wasn't clear enough. The point of:
> So just leaving the comment as a trail.

Was actually not to propose a change, but leave a comment here for future code archeologists that might wonder :)


Indeed changing this to _DSN would require changing the existing values -files and I don't think we want that now.